### ISSUE TITLE
System - Cannot store DOI numbers as aliases

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#359]
The current implementation stores DOI numbers directly as aliases (with '/' converted to '_'); however, the DOI standard allows any UTF-8 characters, but ArangoDB does not allow these characters in document keys. So aliases must be implemented as plain text fields that are indexed. For the time being, the current system will probably work since most DOIs use only a subset of available characters.

**Comment by dvstans** [comment]
We currently have no plans to support publishing. This may change in the future, so will leave this ticket open.

**Comment by JoshuaSBrown** [comment]
I think we should think about what this means. What are we defining as publishing? I think publishing and public sharing are two different things publishing implies static fixed files which I think is outside of the scope of what DataFed is designed for.

**Comment by dvstans** [comment]
Right, publishing is not part of DataFed, but there was a use case where data published from DataFed is left in DataFed for FAIR reasons (no publishing system has the capabilities of DataFed).
